scripts/get_version.kts

```diff
@@ -0,0 +1,15 @@
+object GetVersion {
+
+    private const val PATTERN_VERSION = "[0-9]+.[0-9]+.[0-9]+"
+
+    fun search(commitMessage: String): String? {
+        return PATTERN_VERSION
+            .toRegex()
+            .find(commitMessage)
+            ?.value
+    }
+}
+
+val version = GetVersion.search(commitMessage = args.first())
+
+println(version)
```

Comment:
What if the version is null? Shall we default the empty string in this case?
```suggestion
println(version.orEmpty())
```

Reply:
@lydiasama if `version` is `null`, release-drafter resolves the version automatically by incrementing the _patch_ from the last release.

Reply:
@AVI5HEK I could be wrong, but I think it will print "null" in this case 🙏

Reply:
@AVI5HEK I meant in case `GetVersion.search(commitMessage = args.first())` returns `null`. 🙏🏻

Reply:
@AVI5HEK @Tuubz can we have unit test for this? 🤔

Reply:
That's actually a good idea 👍 

@AVI5HEK Do you think you can add this by end of today? If not, let's skip it for now

Reply:
@Tuubz @luongvo not sure where and how to write the unit test for this. The file is outside of both the projects. Any insight regarding where to begin would be much appreciated. 🙏🏽

Reply:
@Tuubz 
>I think it will print "null" in this case
That's correct. And if it's null, release-drafter will automatically resolve the version. We can edit the version (and tag) from the draft later on.

Reply:
@Tuubz that's what I found while testing on my repo. So if somehow the script fails to get the version from the commit message, i.e, `println(null)`, `release-drafter` will set the version to `3.8.1`, I believe. It incremented the _patch_ value in my case.

Reply:
@AVI5HEK Thank you for your insights. I think we can add tests later as we don't have ideas to do it now 😄